frontend/src/App.js

```diff
@@ -1,4 +1,4 @@
-import appstyle from "./App.module.css"
+import AppClass from "./App.module.css"
```

Comment:
This is not what I meant. Please clarify before working on anything(**In this tenure**)